### PR TITLE
Add "Admission Controller" to the glossary

### PR DESCRIPTION
**Pull request by patrick-east**
This gives a brief description for the admission controller and links
to the main docs for them.

The verbiage is borrowed from the more detailed documentation and has been shortened somewhat to  fit better with the glossary.

### CODE REVIEW

**Comment by kbhawkey** [comment]
hello @patrick-east . Thanks for the PR. 

preview:
https://deploy-preview-15196--kubernetes-io-master-staging.netlify.com/docs/reference/glossary/?extension=true

File: content/en/docs/reference/glossary/admission-controller.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+short_description: >
+  A piece of code that intercepts requests to the Kubernetes API server prior to persistence of the object.
+
+aka:
+tags:
+- extension
+- security
+---
+A piece of code that intercepts requests to the Kubernetes API server prior to persistence of the object.
+
```
Comment (kbhawkey) [inline content/en/docs/reference/glossary/admission-controller.md]: Would it make sense to explain that the admission controllers are configurable on the kube-apiserver?
Do you think you need:
`If any of the controllers reject the request, the entire request is rejected immediately and an error is returned to the end-user.`


What do you think about replacing the link on Line 20 with a link to the admission controllers page as:
`* [List of available Admission controllers](/docs/reference/access-authn-authz/admission-controllers/#what-does-each-admission-controller-do/)`

Comment (patrick-east) [reply content/en/docs/reference/glossary/admission-controller.md]: > Would it make sense to explain that the admission controllers are configurable on the kube-apiserver?

Yea that make sense, I'll add something for that.

>Do you think you need:
>> If any of the controllers reject the request, the entire request is rejected immediately and an >> error is returned to the end-user.

We could reword things if you would like. I think it's important though to explain the second half of what admission controllers are for (first half being that first sentence explaining that they intercept API requests).

> What do you think about replacing the link on Line 20 with a link to the admission controllers page as:
> * `[List of available Admission controllers](/docs/reference/access-authn-authz/admission-controllers/#what-does-each-admission-controller-do/)`

That works for with me, but I'm curious what the reasoning is to jump further down the page? The top couple sections that would get skipped in those docs are pretty good reading for someone trying to learn more about admission controllers.

Comment (kbhawkey) [reply content/en/docs/reference/glossary/admission-controller.md]: What you have seems fine.
If you don't want to change the link name, that is okay. Alternatively, you could make the start of the main paragraph a link, something like:
[Admission controllers](/docs/reference/access-authn-authz/admission-controllers/).

I think the reason to jump to the list of Admission controllers on the page is that the list of controllers represents most of the page. The top sections are equally as important, though. But, the link you have is fine; I am trying to think of a creative way of linking to the page from the glossary.

**Pull request merged**